app/js/controllers/EditCtrl.js

```diff
@@ -658,23 +677,33 @@ KMCMenu.controller('EditCtrl', ['$scope','$http', '$timeout','PlayerData','Playe
 		if ($scope.invalidProps.length > 0){
 			utilsSvc.alert('Save Player Settings','Some plugin features values are invalid. The player cannot be saved.');
 		}else{
+			var savePlayer = function () {
+				PlayerService.savePlayer($scope.playerData).then(function(value) {
+						localStorageService.remove('tempPlayerID'); // remove temp player from storage (used for deleting unsaved players)
+						apiService.setCache(false);                 // prevent the list controller from using the cache the next time the list loads
+						utilsSvc.alert('Save Player Settings','Player Saved Successfully');
+					},
+					function(msg) {
+						utilsSvc.alert('Player save failure',msg);
+					}
+				);
+			};
 			$scope.updatePlayerData();
 			$scope.dataChanged = false;
 			window.parent.studioDataChanged = false; // used when navigating away from studio
 			if ($scope.playerData.config.plugins.playlistAPI && $scope.playerData.config.plugins.playlistAPI.plugin){
 				$scope.addTags(['html5studio','playlist']); // set playlist tag
 			}else{
-				$scope.addTags(['html5studio','player']); // set player tag
+				$scope.addTags(['kalturaPlayerJs','player']); // set player tag
+			}
+			if (!$scope.playerData.autoUpdate && PlayerService.getPlayerVersion($scope.playerData) === '{latest}') {
+				//should load the latest KalturaPlayer to figure out the version number
+				PlayerService.loadKalturaPlayerScript($scope.playerData, function () {
```

Comment:
@yairans - why do you need the player version? Isn't {latest} can be used?

Reply:
in v3 studio you can freeze the version to not be updated automatically. for this we have to extract the version.